smalltalksrc/Slang/TStatementListNode.class.st

```diff
@@ -356,7 +356,9 @@ TStatementListNode >> copyWithoutReturn [
 
 	| newStmtsListNode |
 	self assert: self endsWithReturn.
-	statements size = 1 ifTrue: [ ^ statements last expression ].
+	statements size = 1 ifTrue: [
+		statements last parent: self parent.
```

Comment:
Why this?

Reply:
I will reference this in the first issue I created